docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -331,154 +324,109 @@ using System.Threading.Tasks;
+            "dependencies":{
+                "AwesomeLibrary.Core":"1.0.0"
+            }
+        }
+    }
+}
+```
 
-* Projects are just folders with files in them.
-* There is a `global.json` file at the top level of your library which needs the names of each project (folder name).
-* A project using another project needs to register a dependency in its `project.json` before it can be used.
+And that's it!  To learn more about `project.json` and all of the configuration you can do, check out the [.NET Core project system](../../project-model/overview.md).
 
 ## How to test libraries on .NET Core
 
-**Note: You can only use DNX to test at this time**
+**TODO** this needs to be updated to use dotnet-test-xunit instead of dnx
 
-It's important to be able to test across platforms.  You can do this using [Xunit](http://xunit.github.io/), which is also the testing tool used by .NET Core projects.  Getting a test to work across platforms can be done like this:
+It's important to be able to test across platforms.  You can do this using [Xunit](http://xunit.github.io/), which is also the testing tool used by .NET Core projects.  Setting up your solution with test projects will depending on the [structure of your solution](#structuring-your-solution).  The following example wil assume that all source projects are under a top-level `/src` folder and all test projects are under a top-level `/test` folder.
 
-1. Create a folder for your test project to live in.  It's best to be consistent with your source folder:
+1. Ensure you have a `global.json` file at the solution level which understands where the test projects are:
     
+    ```json
+    "projects":[ "src", "test"]
     ```
-    /src
-       /Library
-    /test
-       /LibraryTests
-    ```
-    
-2. Add a `global.json` file to the root directory of your library.
 
-    ```
-    /src
-       /Library
-    /test
-       /LibraryTests
-    global.json
-    ```
-    
-    And put the names of the src and test folders:
-    
-    ```javascript
-    {
-    	"projects": [
-    		"src", "test"
-    	]
-    }
-    ```
+2. Create a new test project.  Do this by creating a new folder with a `project.json` that has the following:
 
-3. Add a new `project.json` file under your test library folder.  Make sure you have the following:
+**TODO:** Get this correct for RC2, potentially call out dotnet new enhancements.
 
-    * A reference to `xunit` version `2.1.0-*`.
-    * A version to `xunit.runner.dnx` version `2.1.0-rc1-*`.
-    * A reference to the library under test (e.g. "Library" if its namespace is `Library`).
-    * A new command called `test` which references `xunit.runner.dnx`.
+    * A reference to xUnit version `2.1.0` and the xUnit DNX test runner version `2.1.-rc1-*` under `dependencies`.
+    * A reference to the library under test (e.g. "LibraryUnderTest" if the name of the library is `LibraryUnderTest`).
+    * A new dnx command called `test` which references `xunit.runnder.dnx`.
```

Comment:
I'll be removing this section.  Right now I'm still having trouble getting packages to restore with xUnit, but this will no longer have DNX.